js/utils.js

```diff
@@ -394,28 +394,33 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
 
     return _.filter(records, function (record) {
       return _.every(filters, function (filter) {
-        if (filter.condition === 'empty') {
-          return _.isEmpty(record.data[filter.column]);
-        }
-          
-        if (filter.condition === 'notempty') {
-          return !_.isEmpty(record.data[filter.column]); 
-        }
+        var condition = filter.condition;
+        var rowData = _.get(record, 'data.' + filter.column, null);
```

Comment:
Using a string path ends up using `String.prototype.split()` which does not perform very well. This isn't usually a problem, but on an operation like this where the operation is executed over a potentially very large list, it's more efficient to pass an array as path.

```suggestion
        var rowData = _.get(record, ['data', filter.column], null);
```

Reply:
Fixed